package.json

```diff
@@ -65,7 +65,7 @@
     "@angular/platform-browser": "^7.2.13",
     "@angular/platform-browser-dynamic": "^7.2.13",
     "@angular/router": "^7.2.13",
-    "@hmcts/ccd-case-ui-toolkit": "^2.67.3-tab-name-replaceAll",
+    "@hmcts/ccd-case-ui-toolkit": "v2.68.6-non-descriptive-headings",
```

Comment:
v?

Reply:
Removed unnecessary v (although it appeared to work with it in there)